docs/run.md

```diff
@@ -10,7 +10,10 @@ go run github.com/onosproject/onos-config/cmd/onos-config \
     -configStore=$HOME/go/src/github.com/onosproject/onos-config/configs/configStore-sample.json \
     -changeStore=$HOME/go/src/github.com/onosproject/onos-config/configs/changeStore-sample.json \
     -deviceStore=$HOME/go/src/github.com/onosproject/onos-config/configs/deviceStore-sample.json \
-    -networkStore=$HOME/go/src/github.com/onosproject/onos-config/configs/networkStore-sample.json
+    -networkStore=$HOME/go/src/github.com/onosproject/onos-config/configs/networkStore-sample.json \
+    -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/TestDevice-1.0.0/testdevice.so.1.0.0 \
+    -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/TestDevice-2.0.0/testdevice.so.1.0.0 \
```

Comment:
can we add just the emulator one ? the test device is not really present. Useful for testing though.

Reply:
Individual users can add what they want when they run onos-config - I think it will be important to include all for testing purposes. I will also add a gRPC to manage these plugins

Reply:
ok, I agree.